docker/maintenance/cronjob-generator.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+## usage ./cronjob-generator
+SCRIPT_FOLDER=$(dirname $(readlink -f $0))
+JOB_DESCRIPTIONS=(`ls $SCRIPT_FOLDER | grep -v 'example' | grep -v 'cronjob-template.yaml' | grep '.yaml'`)
```

Comment:
instead of filtering this way you can pass list of files as an argument
what that means you don't need to make a script that handle all files in iteration, but script that handle one file, 
then you can encapsulate this line in other file that call `handle one file` script

Reply:
You can at least filter out YAML-less ;) files at ls level - `ls $SCRIPT_FOLDER *.yaml` instead of later `grep`.

Reply:
actually that won't work
as ls does not have any filter option
if you do 
ls *.yaml
you're actually using globbing of your shell and pass it to ls for formatting only 

so to make it work
we'd have to 
ls $SCRIPT_FOLDER/*.yaml

but then it gives me file names with full paths that I'd have to clean there
so I think it's easier to leave it as is